helm/net-exporter/templates/daemonset.yaml

```diff
@@ -20,14 +20,22 @@ spec:
     spec:
       initContainers:
       - name: label-kube-system-namespace
+        {{ if (.Values.Installation) }}
+        image: "{{ .Values.Installation.V1.Registry.Domain }}/{{ .Values.namespaceLabeler.image.name }}"
+        {{ else }}
         image: "{{ .Values.image.registry }}/{{ .Values.namespaceLabeler.image.name }}"
+        {{ end }}
         securityContext:
           runAsUser: 1000
           runAsGroup: 1000
       priorityClassName: giantswarm-critical
       containers:
       - name: net-exporter
-        image: "{{ .Values.image.registry }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
+        {{ if (.Values.Installation) }}
+        image: "{{ .Values.Installation.V1.Registry.Domain }}/{{ .Values.image.name }}::{{ .Values.image.tag }}"
```

Comment:
@calvix both if and else branch have a bug in image reference, `::` should be just one `:`. I'm testing removal of CPU limits and uncovered this there, will fix it.